WGS_pipeline/opencravat.wdl

```diff
@@ -4,87 +4,52 @@ version 1.0
 # more information available at https://open-cravat.readthedocs.io/en/latest/2.-Command-line-usage.html
```

Comment:
is this wdl still being used?

Reply:
not really but it is a more general version than the depmap one. so let's keep it available for people wanting to use a simpler version.